b2sdk/transfer/inbound/downloader/range.py

```diff
@@ -26,7 +26,7 @@ def from_header(cls, raw_range_header):
 
         raw_range_header example: 'bytes 0-11'
         """
-        offsets = tuple(int(i) for i in raw_range_header.replace('bytes ', '').split('-'))
+        offsets = tuple(int(i) for i in raw_range_header.replace('bytes=', '').split('-'))
```

Comment:
separate PR to explain why?

Reply:
because it is a bug? :D :P it seems that it is just not working now :) but probably no-one ever tried to download a file range - and it was a basic case in this PR so I had to fix it to be able to test - but yes - we can fix it on master right now:
```
    @classmethod
    def _get_remote_range(cls, response, metadata):
        """
        Get a range from response or original request (as appropriate).

        :param response: requests.Response of initial request
        :param metadata: metadata dict of the target file
        :return: a range object
        """
        raw_range_header = response.request.headers.get('Range')  # 'bytes 0-11'
        if raw_range_header is None:
            return Range(0, 0 if metadata.content_length == 0 else metadata.content_length - 1)
        return Range.from_header(raw_range_header)
```
I think the problem is "from response or original request (as appropriate)." - it actually only get it from original request... but... https://tools.ietf.org/html/rfc7233#section-2.1
```
   Examples of byte-ranges-specifier values:

   o  The first 500 bytes (byte offsets 0-499, inclusive):

        bytes=0-499

   o  The second 500 bytes (byte offsets 500-999, inclusive):

        bytes=500-999
```